hal/transport/RS485/MyTransportRS485.cpp

```diff
@@ -154,6 +154,12 @@ bool _serialProcess()
 				}
 				_recPhase = 1;
 				_recPos = 0;
+				
```

Comment:
You have trailing whitespaces here.
That is what cause the butler to attempt to mail you a patch to remove them.